src/ast2ram/ProvenanceClauseTranslator.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Souffle - A Datalog Compiler
+ * Copyright (c) 2013, 2015, Oracle and/or its affiliates. All rights reserved
```

Comment:
This date must be more recent. 2018+, I think. It's also by the Souffle Developers.